swo/mpt/cli/core/products/app.py

```diff
@@ -0,0 +1,115 @@
+from typing import Annotated, Optional
+
+import typer
+from rich import box
+from rich.table import Table
+from swo.mpt.cli.core.accounts.flows import (
+    find_active_account,
+    get_accounts_file_path,
+    get_or_create_accounts,
+)
+from swo.mpt.cli.core.console import console
+from swo.mpt.cli.core.errors import NoActiveAccountFoundError
+from swo.mpt.cli.core.mpt.client import client_from_account
+from swo.mpt.cli.core.mpt.flows import get_products
+from swo.mpt.cli.core.mpt.models import Account as MPTAccount
+from swo.mpt.cli.core.mpt.models import Product as MPTProduct
+
+app = typer.Typer()
+
+
+DEFAULT_LIMIT = 10
+
+
+def _limit_offset_by_page(page: int) -> tuple[int, int]:
+    return (
+        DEFAULT_LIMIT,
+        (page - 1) * DEFAULT_LIMIT,
+    )
+
+
+@app.command(name="list")
+def list_products(
+    page: Annotated[
+        int,
+        typer.Option("--page", "-p", help="Products page number"),
```

Comment:
What we where discussing is product page size not retrieving a given page number :D